### PR TITLE
GzBuilder methods now take Into<Vec<u8>>.

### PR DESCRIPTION
**Pull request by Matt8898**
Solves issue #84.

### CODE REVIEW

File: src/gz.rs [hunk 1]
```diff
@@ -998,10 +998,10 @@ mod tests {
 
     #[test]
     fn fields() {
-        let r = vec![0, 2, 4, 6];
+		let r = vec![0, 2, 4, 6];
         let e = Builder::new()
-                    .filename(b"foo.rs")
-                    .comment(b"bar")
+                    .filename(&b"foo.rs"[..])
+                    .comment(&b"bar"[..])
```
Comment (alexcrichton) [inline src/gz.rs]: Hm I think that this works if the `b` is removed here, right?

File: src/gz.rs [hunk 2]
```diff
@@ -998,10 +998,10 @@ mod tests {
 
     #[test]
     fn fields() {
-        let r = vec![0, 2, 4, 6];
+		let r = vec![0, 2, 4, 6];
```
Comment (alexcrichton) [inline src/gz.rs]: Looks like some extra tabs may have snuck in?

**Comment by alexcrichton** [comment]
Thanks @Matt8898!

**Comment by Matt8898** [comment]
Yes, that works without b.

**Comment by Matt8898** [comment]
I'm going to change that as soon as possible.

**Pull request closed without merging**